packages/components/src/Badge/Badge.scss

```diff
@@ -343,4 +348,14 @@ $tc-badge-disabled-opacity: 0.62;
 	&--value {
 		@include colored-badge(tint($lochmara, 70%), shade($lochmara, 30%));
 	}
+
+	&-dropdown {
+		.tc-badge-button {
+			&:not(.tc-badge-disabled) {
+				&:global(.ally-focus-within) {
+					border-color: rgba($lochmara, 1);
```

Comment:
why using rgba here?

Reply:
actually not required, but just using this to be in sync with the other badge types
https://github.com/Talend/ui/blob/4b3e2917ba645601781c8eac40e439adf7481973/packages/components/src/Badge/Badge.scss#L300

Reply:
no, please remove if you don't adjust alpha 🙏

Reply:
fixed